web-app/src/app/tabs/controls/controls.component.html

```diff
@@ -1,3 +1,123 @@
-<p>
-  controls works!
-</p>
+<div class="controls">
+	<div class="header">
+		<span class="aliveGroup">
+	    		<div *ngIf="controls.alive; else controlsNotAlive"><img class="circleAlive" src="../../../assets/GreenCircle.png" alt="Red Circle" id="circle"></div>
+	    		<ng-template #controlsNotAlive><img class="circleAlive" src="../../../assets/RedCircle.png" alt="Red Circle" id="circle"></ng-template>
+				<span class="aliveText">Alive</span>
+		</span>
+		<span class="aliveGroup">
+	    		<div *ngIf="lights.alive; else elseBlock"><img class="circleAlive" src="../../../assets/GreenCircle.png" alt="Red Circle" id="circle"></div>
```

Comment:
We only need 1 block for the green circle, so `#controlsNotAlive` isn't needed.

Also, change "Red Circle" to green circle. 

All instances of `id="circle"` should be dropped.